src/containers/Event/Document/Document.js

```diff
@@ -1,16 +1,125 @@
-import React from "react";
+import React, { useState, useContext, useEffect } from "react";
+import {
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  IconButton,
+  Tooltip,
+  Collapse,
+} from "@material-ui/core";
+
+import {
+  Delete as DeleteIcon,
+  AddCircle as AddCircleIcon,
+} from "@material-ui/icons";
+
+import AddDoc from "./AddDoc";
 import "./Document.css";
+import { store } from "../../../store.js";
 
 function Document(props) {
+  const { currentUser, docs, deleteDoc } = useContext(store);
+  const [eventDocs, setEventDocs] = useState(docs);
+  const [docId, setDocId] = useState("");
+  const [docSrc, setDocSrc] = useState("");
```

Comment:
You don't need both `docId` and `docSrc`, use only `docId`

Reply:
This should be resolved now.